src/array/array.cpp

```diff
@@ -227,9 +227,9 @@ Array::Delete(void)
         goto error;
     }
 
+    _ClearPbr();
     _DeletePartitions();
     devMgr_->Clear();
-    _ClearPbr();
```

Comment:
device 목록 clear전에 pbr을 먼저 clear

Reply:
지금 말씀 주신 내용을 코멘트로 남기는 것이 어떨까요?? 배경이 충분치 않은 상태에서는, `_ClearPbr()`과 `devMgr_->Clear()` 간의 순서가 중요하다는 것을 인지하지 못할 가능성이 클 것 같아서요, 
```c++
_ClearPbr(); // this must come before devMgr_->Clear(), to avoid dangling-pointers. 
```
(혹시 설명이 적절치 않으면 코멘트 수정하셔도 좋을 것 같습니다!)

Reply:
@yyu-nim 님 코멘트 반영하시는게 좋을 것 같습니다. 아니면 state를 나눠서 처리 하도록 강제하는 방법도 있을 것 같구요

Reply:
네 이후 이와 연결된 후속 커밋에서 반영하겠습니다.